rewrite-gradle/src/test/java/org/openrewrite/gradle/plugins/RemoveBuildPluginTest.java

```diff
@@ -99,7 +99,8 @@ void removeUnappliedPluginWithoutVersion() {
     void applySyntax() {
         rewriteRun(
           spec -> spec.recipe(new RemoveBuildPlugin("org.openrewrite.rewrite")),
-          buildGradle("""
+          buildGradle(
+                """
```

Comment:
Looks like the indentation is still a bit off (6 instead of just 2).

Reply:
Hmm; doesn't seem feasible to resolve from IDE, given the number of files and changes; even a quick attempt with IntelliJ then throws up more unrelated changes, so then perhaps best not to apply this one here just yet.